crates/network-scheduler/Cargo.toml

```diff
@@ -44,6 +44,9 @@ sqd-contract-client = { workspace = true }
 sqd-messages = { workspace = true, features = ["semver"] }
 sqd-network-transport = { workspace = true, features = ["scheduler", "metrics"] }
 chrono = "0.4.38"
+bs58 = "0.5.1"
+crypto_box = "0.9.1"
+curve25519-dalek = "4.1.3"
```

Comment:
Dependencies should be sorted alphabetically

Reply:
sorted